sql/mysql_db/replica_source_info.go

```diff
@@ -0,0 +1,128 @@
+// Copyright 2023 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mysql_db
+
+import (
+	"encoding/json"
+
+	"github.com/dolthub/go-mysql-server/sql"
+	"github.com/dolthub/go-mysql-server/sql/in_mem_table"
+)
+
+// ReplicaSourceInfo represents the binlog replication metadata persisted
+// in the mysql database.
+// For more details, see: https://dev.mysql.com/doc/refman/8.0/en/replica-logs-status.html
+type ReplicaSourceInfo struct {
+	Host     string
+	User     string
+	Password string
+	Port     uint16
+	Uuid     string
+}
+
+var _ in_mem_table.Entry = (*ReplicaSourceInfo)(nil)
+
+// NewFromRow implements the interface in_mem_table.Entry.
+func (r *ReplicaSourceInfo) NewFromRow(_ *sql.Context, row sql.Row) (in_mem_table.Entry, error) {
+	if err := userTblSchema.CheckRow(row); err != nil {
```

Comment:
Should be `replicaSourceInfoTblSchema` right?

Reply:
ooooh... good catch! Thank you!